src/OpenGL/MonoMacGameView.cs

```diff
@@ -120,24 +122,36 @@ public override void DrawRect (CGRect dirtyRect)
 		public override void LockFocus ()
 		{
 			base.LockFocus ();
-			if (openGLContext.View != this)
-				openGLContext.View = this;
+			if (ActualOpenGLContext.View != this)
+				ActualOpenGLContext.View = this;
 		}
 
-		public NSOpenGLContext OpenGLContext {
-			get { return openGLContext; }
+		public NSOpenGLContext? OpenGLContext {
+			get => openGLContext;
+		}
+
+		public NSOpenGLContext ActualOpenGLContext {
```

Comment:
This should not be public, you are adding a public property that is not needed since we have the OpenGLContext. This is an implementation detail that we are leaking to the users.
```suggestion
		private  NSOpenGLContext ActualOpenGLContext {
```